seq2science/rules/get_fastq.smk

```diff
@@ -43,21 +44,26 @@ rule run2sra:
             ) 200>{PYSRADB_CACHE_LOCK}
 
             # dump
-            prefetch --max-size 999999999999 --output-directory ./ --log-level debug --progress {wildcards.run} >> {log} 2>&1 && break
+            prefetch --max-size u --output-directory ./ --log-level debug --progress {wildcards.run} \
+            >> {log} 2>&1 && break
+            
+            # retry
+            echo "DEBUG: prefetch try ${{i}} of 3 failed" >> {log}
             sleep 10
         done
 
-        # TODO: this is the strangest bug, in that on some machines (ocimum) prefetch downloads
-        # to a different location. Not sure what causes this, but this should fix that. Could simply
-        # be a global setting that we haven't discovered yet...
+        # TODO: section can be removed if we dont see the debug message
         # bug report: https://github.com/ncbi/sra-tools/issues/533
         if [[ -f "{params.outdir}/{wildcards.run}.sra" ]]; then
+            echo "DEBUG: moving output to correct directory" >> {log}
             mkdir -p {params.outdir}/{wildcards.run}
             mv {params.outdir}/{wildcards.run}.sra {output}
         fi
 
+        # TODO: section can be removed if we dont see the debug message
         # If an sralite file was downloaded instead of a sra file, just rename it
         if [[ -f "{params.outdir}/{wildcards.run}.sralite" ]]; then
+            echo "DEBUG: renaming SRAlite" >> {log}
             mkdir -p {params.outdir}/{wildcards.run}
             mv {params.outdir}/{wildcards.run}.sralite {output}
         fi
```

Comment:
I wouldn't be so sure that just because we don't have these issues with our test data, that we don't get them in general. The sralite one used to be specific to some accessions if I remember correctly

Reply:
the top issue should be fixed in prefetch now. I left it in because I agree ;)